zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -338,7 +338,11 @@ public void removeNote(String id, AuthenticationInfo subject) {
       note = notes.remove(id);
       folders.removeNote(note);
     }
-    interpreterSettingManager.removeNoteInterpreterSettingBinding(subject.getUser(), id);
+    try {
+      interpreterSettingManager.removeNoteInterpreterSettingBinding(subject.getUser(), id);
+    } catch (IOException e) {
+      logger.error(e.toString(), e);
```

Comment:
Your opinion is worth thinking about.
However, the intent of this fix is to make the build successful,
It is to unify the code of the master branch and branch-0.7.
Therefore, you can not modify it in this PR.
Sorry